contracts/Staking.sol

```diff
@@ -90,11 +90,10 @@ contract Staking {
     /*
      * Removes the deposit of the user and sends the amount of `tokenAddress` back to the `user`
      */
-    function withdraw(address tokenAddress) public {
-        require(balances[msg.sender][tokenAddress] > 0, "Staking: User has empty balance");
+    function withdraw(address tokenAddress, uint256 amount) public {
```

Comment:
Maybe it should be a good idea to have both methods, the second one being an overload to the normal withdraw

Reply:
I think it would be confusing for the users. If we really want that, I'd call it something different, like `withdrawFull` or something like that.

Reply:
It would duplicate a lot of code imo.

Reply:
We could have a withdraw(address tokenAddress) that calls withdraw(address tokenAddress, uint256 amount) with the full amount, and not duplicate any code 👍

Reply:
Either way, I think that is outside the scope of this PR. If we decide to implement that, we can do it in another PR